src/goose.rs

```diff
@@ -1245,6 +1245,7 @@ impl GooseUser {
         // POST request.
         let goose_request = GooseRequest::builder()
             .method(GooseMethod::Post)
+            .path(path)
```

Comment:
We have to call `.path(path)` for consistent naming in the metrics, for this to work the same if you're using the builder directly.